src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+  const threeDays = 3;
+  const sevenDays = 7;
+  const twentyOff = 20;
+  const fifteenOff = 50;
```

Comment:
it's better to add names that can be appropriate for different values
```suggestion
  const rentalCost = 40 * days
  const daysForBasicDiscount = 3;
  const daysForProDiscount = 7;
  const basicDiscount = 20;
  const proDiscount = 50;
```